R/annotate_fusion_calls.R

```diff
@@ -3,10 +3,11 @@
 #' @param standardFusioncalls A dataframe from star fusion or arriba standardized to run through the filtering steps
 #' @param geneListReferenceDataTab A dataframe with column 1 as GeneName 2 source file 3 type; collapse to summarize type
 #' @param fusionReferenceDataTab A dataframe with column 1 as FusionName 2 source file 3 type; collapse to summarize type
-#'
+#' @param checkReciprocal Logical value to check if fusion also has reciprocal fusion in Sample, default to FALSE
```

Comment:
```suggestion
#' @param checkReciprocal Logical value to check if fusion also has reciprocal fusion in Sample, default to TRUE
```

@kgaonkar6 I think this should be a default == T